lib/operations/command.js

```diff
@@ -44,39 +54,44 @@ class CommandOperation extends OperationBase {
     }
   }
 
+  _buildCommand() {
```

Comment:
Consider removing this method so that we catch subclasses that forget to implement it.

Reply:
What if we made it throw with an explicit error, so that we would get more detail than a `TypeError: x._buildCommand is not a function`?

Reply:
fine by me

Reply:
We need this for raw commands.